libraries/package-deps-hash/src/getPackageDeps.ts

```diff
@@ -2,10 +2,41 @@
 // See LICENSE in the project root for license information.
 
 import * as child_process from 'child_process';
+import * as path from 'path';
 import { Executable } from '@rushstack/node-core-library';
 
 import { IPackageDeps } from './IPackageDeps';
 
+/**
+ * Parses a quoted filename sourced from the output of the "git status" command.
+ *
+ * Paths with non-standard characters will be enclosed with double-quotes, and non-standard
+ * characters will be backslash escaped (ex. double-quotes, non-ASCII characters). The
+ * escaped chars can be included in one of two ways:
+ * - backslash-escaped chars (ex. \")
+ * - octal encoded chars (ex. \347)
+ *
+ * See documentation: https://git-scm.com/docs/git-status
+ */
+export function parseGitFilename(filename: string): string {
+  // If there are no double-quotes around the string, then there are no escaped characters
+  // to decode, so just return
+  if (!filename.match(/^".+"$/)) {
```

Comment:
What if the input is `"exmaple" ` with a trailing space?  Is that a realistic input?  If not should we report an error or `trim()`?

Reply:
I'd say that this isn't really an issue. Git output ends the line immediately after the closing quote. I think if this started showing up, it would be new behaviour and we'd need to modify the code here anyway